chapter_tidy/chapter_tidy.html

```diff
@@ -8,8 +8,8 @@
     <link href="libs/countdown/countdown.css" rel="stylesheet" />
     <script src="libs/countdown/countdown.js"></script>
     <script src="https://use.fontawesome.com/5235085b15.js"></script>
-    <link rel="stylesheet" href="../css/scpo.css" type="text/css" />
-    <link rel="stylesheet" href="../css/scpo-fonts.css" type="text/css" />
+    <link rel="stylesheet" href="C:/Users/mylen/Dropbox/Teaching/Metrics/ScPoEconometrics-Slides/css/scpo.css" type="text/css" />
```

Comment:
this will not work on somebody else's computer... 
did you change this manually? it should always have the relative file path

Reply:
Yes we saw this, I'm fixing it :)